modules/cdp-engine/src/lib.rs

```diff
@@ -87,7 +88,7 @@ pub struct RiskManagementParams {
 	pub maximum_total_debit_value: Balance,
 
 	/// Extra interest rate per sec, `None` value means not set
-	pub interest_rate_per_sec: Option<Rate>,
+	pub interest_rate_per_sec: Option<FractionalRate>,
```

Comment:
the bound and change limit check of `FractionalRate` doesn't make much sense for the `interest_rate_per_sec` of `RiskManagementParams`,  the config value is too tiny(for example 0.000000000860244)

Reply:
We can definitely further limit the range, if reasonable bounds can be defined. Any suggestion for the upper limit of `interest_rate_per_sec`?

Reply:
we can define reasonable bound for system params in #2362